src/docbuild/cli/cmd_validate/process.py

```diff
@@ -209,7 +246,26 @@ async def process_file(
     # IDEA: Should we replace jing and validate with etree.RelaxNG?
 
     # 1. RNG Validation
-    rng_success, rng_output = await validate_rng(path_obj)
+    validation_method = getattr(context, 'validation_method', 'jing')
```

Comment:
It works, but as you already set the `validation_method` attribute to `jing`, we can simplify it and be more explicit:

```suggestion
    validation_method = context.validation_method
```